EleksTubeHAX/EleksTubeHAX.ino

```diff
@@ -332,14 +332,21 @@ void setupMenu() {
   tfts.setCursor(0, 124, 4);
 }
 
+bool isDimmingTime(uint8_t current_hour) {
+    if (DAY_TIME < NIGHT_TIME)
+      return (current_hour >= NIGHT_TIME) || (current_hour < DAY_TIME);
+    else
+      return (NIGHT_TIME <= current_hour) && (current_hour < DAY_TIME);  
+}
+
 void EveryFullHour() {
   // dim the clock at night
   uint8_t current_hour = uclock.getHour24();
   FullHour = current_hour != hour_old;
   if (FullHour) {
   Serial.print("current hour = ");
   Serial.println(current_hour);
-    if ((current_hour >= NIGHT_TIME) || (current_hour < DAY_TIME)) {
+    if (isDimmingTime(current_hour)) {
```

Comment:
Changed the name of the function. If you accept the above, you need to accept this one too.
```suggestion
    if (isNightTime(current_hour)) {
```